app/Http/Controllers/Licenses/LicenseCheckinController.php

```diff
@@ -106,4 +106,44 @@ public function store(Request $request, $seatId = null, $backTo = null)
         // Redirect to the license page with error
         return redirect()->route('licenses.index')->with('error', trans('admin/licenses/message.checkin.error'));
     }
+
+    public function checkinCount($license_id){
+        $licenseSeats = LicenseSeat::where('license_id', '=', $license_id)
+            ->whereNotNull('assigned_to')
+            ->with('user')
+            ->get();
+
+        return('There are '.$licenseSeats->count().' seats checked out. Are you sure you want to check all of them in?');
+    }
+
+
+    public static function checkinLicense($license_id){
+
+        $licenseSeats = LicenseSeat::where('license_id', '=', $license_id)
+            ->whereNotNull('assigned_to')
+            ->with('user')
+            ->get();
+
+        if (! $license_id) {
+            return redirect()->to('/licenses')->with('error', 'License does not exist.');
+
+        }
+
+        if (!License::where('id', '=', $license_id)->first()) {
+            return redirect()->to('/licenses')->with('error', 'Invalid license ID.');
+        }
+
+        foreach ($licenseSeats as $seat) {
+            $seat->assigned_to = null;
+
+            if ($seat->save()) {
+                // Override the email address so we don't notify on checkin
+                    $seat->user->email = null;
```

Comment:
Do we not want to notify on checkin? That should be determined like the category, like we normally do, no?

Reply:
I'm not sure? This is what we have been doing in the console command version